lib/functions.php

```diff
@@ -331,7 +331,7 @@ function parseMimeType(string $str): array
     if (2 !== count($mimeType)) {
         // Illegal value
         var_dump($mimeType);
```

Comment:
This looks suspicious

Reply:
Yes, this code has been sitting there for "a long time".
See comment from last year: https://github.com/sabre-io/http/pull/176#pullrequestreview-795621361

IMO this can/should be fixed like proposed in https://github.com/sabre-io/http/pull/98/files - I just did not get an answer to my comment on #176 

We can fix this in this PR, which will have a major version release anyway - although I can't think why someone would want that existing var_dump-and-exit behavior!

Reply:
I  got rid of this now.